Fix descriptions in Beatmap Editor (index page) (#2753)
ppy/osu-wiki: wiki/Beatmap_Editor/en.md

# Beatmap Editor

> The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It's separated into four parts:

Reviewer: ```suggestion
The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
```

- [Compose](Compose): The place where a majority of beatmapping happens.
- [Design](Design): The editor for where Storyboards are made; however, most mappers only use the "Background Image/Video" button here.

> - [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hitsounds, and the hitsound volume.

Reviewer: ```suggestion
- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hit sounds, and the hit sound volume.
```

> - [Song Setup](Song_Setup): The first place of where all new beatmaps begin. This is sets the values regarding the title, artist, version name, difficulty, combo colours, and more.

Reviewer: ```suggestion
- [Song Setup](Song_Setup): The first place of where all new beatmaps begin. This sets the values regarding the title, artist, version name, difficulty, combo colours, and more.
```
